resources/views/admin/checkout.blade.php

```diff
@@ -13,8 +13,153 @@
         <div class="card">
             <div class="card-content">
                 <span class="card-title">@lang('checkout.checkout')</span>
+                <blockquote>
```

Comment:
Same as above, please open an issue to remove these code duplications and create a blade template.